architecture/networking/routes.adoc

```diff
@@ -296,8 +296,10 @@ Path based routes specify a path component that can be compared against
 a URL (which requires that the traffic for the route be HTTP based) such
 that multiple routes can be served using the same host name, each with a
 different path. Routers should match routes based on the most specific
-path to the least; however, this depends on the router implementation. The
-following table shows example routes and their accessibility:
+path to the least; however, this depends on the router implementation. 
+The host name and path are passed through to the backend server so it should be
+able to successfully answer requests for them. 
```

Comment:
Can we make it abundantly clear that a request to http://example.com/foo/ that goes to the router will result in the pod seeing a request to http://example.com/foo/.

Perhaps add:

For example a request to http://example.com/foo/ that goes to the router will result in a pod seeing a request to http://example.com/foo/.

(with the right formatting)

Reply:
thanks, i'll add this changes